README.md

```diff
@@ -112,6 +112,50 @@ You should see a [Hardhat simulation](https://hardhat.org/hardhat-runner/docs/ge
 sequentially processing all available updates. At the time of this writing, each
 update costs around 330K in gas.
 
+### Run relayer
+
+To run the relayer you need to have a redis instance running
```

Comment:
The README section should start with instructions how to run the relay as a docker image. We'll use the [Github docker registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-docker-registry) to publish the image, so anyone can run it with a single `docker run` command.